flask_resty/api.py

```diff
@@ -28,6 +27,12 @@ def handle_api_error(error):
     def _get_app(self, app):
         app = app or self._app
         assert app, "no application specified"
+        # store the view urls for reference. Doesn't support multiple routes
+        # mapped to same view
+        app.extensions.setdefault('resty', {
```

Comment:
You'd put the `views={}` into the ctor, but otherwise yeah. No need to prepend with underscore; we're not exporting that class anyway.